cmd/customd/client/client.go

```diff
@@ -0,0 +1,448 @@
+package client
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/app"
+	"github.com/iov-one/weave/x/sigs"
+	"github.com/pkg/errors"
+	cmn "github.com/tendermint/tendermint/libs/common"
+	tmpubsub "github.com/tendermint/tendermint/libs/pubsub"
+	"github.com/tendermint/tendermint/rpc/client"
+	ctypes "github.com/tendermint/tendermint/rpc/core/types"
+	tmtypes "github.com/tendermint/tendermint/types"
+)
+
+type Header = tmtypes.Header
+type Status = ctypes.ResultStatus
+type GenesisDoc = tmtypes.GenesisDoc
```

Comment:
I am not sure if aliasing types like this is a good thing. It hides where the code comes from and you can no longer use the original type without casting.